tests/lib/rules/no-wait-for-side-effects.test.ts

```diff
@@ -121,7 +197,7 @@ ruleTester.run(RULE_NAME, rule, {
           fireEvent.keyDown(input, {key: 'ArrowDown'})
         })
       `,
-      errors: [{ messageId: 'noSideEffectsWaitFor' }],
+      errors: [{ line: 3, column: 15, messageId: 'noSideEffectsWaitFor' }],
```

Comment:
I'm thinking about making `line` and `column` required on our rule tester. Otherwise, just checking the `messageId` can lead to brittle error reporting.